README.md

```diff
@@ -1,11 +1,9 @@
 # Employee Recognition Platfrom
 
-### Setup
-```
-bin/setup
-```
+### How to access to the website:
 
-### How to start the app
-```
-rails server
-```
+https://employee-recognition-platfrom.herokuapp.com
+
+### If you want to login please use this account: 
+- login: jedynka@gmail.com 
+- password: 111111
```

Comment:
IMO you're forcing someone to count `1`s. WDYT about setting it to `password`?

Reply:
It will definitely make logging in easier for the person who enters for the first time. I changed the data according to your suggestion